tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# cmake-format: off
```

Comment:
why we disable `cmake-format` for the license?

Reply:
It will mess these next 2 lines. Check the first 4 lines (we dont disable the check fo r the entire file!

Reply:
TLDR: the standard says to have a single line for license, the formatter will split long lines and we don't want it.

Reply:
gotcha, thanks for the explanation